src/foam/nanos/notification/email/DAOResourceLoader.java

```diff
@@ -95,6 +101,9 @@ public static EmailTemplate findTemplate(X x, String name, String groupId, Strin
     if ( emailTemplate == null )
       emailTemplate = findTemplateHelper(x, name, "", "en", "", templateArgs);
 
+    if ( emailTemplate == null )
```

Comment:
Why is this necessary? Doesn't findTemplateHelper() do this same check as its last step?